README.md

```diff
@@ -35,7 +35,7 @@ Installing and building:
 
 Running:
 
-    $ prom2json http://my-prometheus-client.example.org:8080/metrics
+    $ prom2json -utl=http://my-prometheus-client.example.org:8080/metrics
```

Comment:
`-url`, perhaps, but I did not think the ask was to change the original parameter to be a flagged argument, only the new ones.